lib/routes/account.js

```diff
@@ -922,7 +966,9 @@ module.exports = function (
             // There's code in the handler that checks for a valid email,
             // no point adding overhead by doing it again here.
             email: isA.string().required(),
-            verified: isA.boolean().required()
+            verified: isA.boolean().required(),
+            sessionVerified: isA.boolean().required(),
```

Comment:
In the `/account/login` endpoint, `verificationMethod` and `verificationReason` are returned. This allows the UI to determine which of the two "confirm your email" screens should be displayed to the user. 

`/recovery_email/status` is called if the user has an existing sessionToken. The response is used in an identical fashion to `/account/login`, to determine which screen should be displayed to the user. Before this PR, if `verified` was false, we'd send the user to the "confirm your email" screen for signup. Here, we might want to display the "confirm this signin" screen instead. 

`sessionVerified` and `emailVerified` can be converted to `verificationReason`, but in the future if we have more than one `verificationMethod`, we won't know which method to use.

I propose the same two values are returned here as for `/account/login` - `verificationMethod` and `verificationReason`. Since `email` is the only possible method, that could be hard coded for now.